packages/k8s.contrib.crd/tests/ModuleGenerator.pkl

```diff
@@ -28,11 +28,11 @@ local generator = (generate) {
   source = "dummy://test_uri"
   converters {
     ["restateclusters.restate.dev"] {
-      [List("spec", "compute", "env", "env")] = EnvVar
+      [List("spec", "compute", "env", "[]")] = EnvVar
       [List("spec", "compute", "resources")] = ResourceRequirements
-      [List("spec", "security", "networkPeers", "ingress", "ingres")] = NetworkPolicy.NetworkPolicyPeer
-      [List("spec", "security", "networkPeers", "admin", "admin")] = NetworkPolicy.NetworkPolicyPeer
-      [List("spec", "security", "networkPeers", "metrics", "metric")] = NetworkPolicy.NetworkPolicyPeer
+      [List("spec", "security", "networkPeers", "ingress", "[]")] = NetworkPolicy.NetworkPolicyPeer
+      [List("spec", "security", "networkPeers", "admin", "[]")] = NetworkPolicy.NetworkPolicyPeer
+      [List("spec", "security", "networkPeers", "metrics", "[]")] = NetworkPolicy.NetworkPolicyPeer
```

Comment:
I needed a way to disambiguate "this is the element of a listing" when resolving naming conflicts.

Previously, the code generator was just appending the same name twice, for "element of a listing", but that can be confused with a normal nested property.

Instead, this represents the listing element as `"[]"`.